src/main/java/org/example/OccurrenceWordPoc.java

```diff
@@ -0,0 +1,56 @@
+package org.example;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class OccurrenceWordPoc {
+    static String inputFilePath;
```

Comment:
Don't need static variables. Just use local variables.

Reply:
I have changed from static to local variables.